packs/gsuite_reports.yml

```diff
@@ -21,6 +21,7 @@ PackDefinition:
     - GSuite.SuspiciousLogins
     - GSuite.TwoStepVerification
     - GSuite.UserSuspended
+    - Google.Workspace.Admin.Custom.Role
```

Comment:
well, this pack's `Description:` says that it is GSuite Reports based, and the contents look to be more generally GSuite. Can you please update the Description to be `Panther GSuite Detections`?  I can cut a PR to bring in missing detections to the pack

Reply:
changing in next commit!